app/src/ui/toolbar/branch-dropdown.tsx

```diff
@@ -159,15 +159,17 @@ export class BranchDropdown extends React.Component<
     let progressValue: number | undefined = undefined
 
     if (checkoutProgress) {
-      title = checkoutProgress.targetBranch
-      description = __DARWIN__ ? 'Switching to Branch' : 'Switching to branch'
+      title = checkoutProgress.target
+      description = __DARWIN__
+        ? 'Checking out the Target'
```

Comment:
Prefer to keep this more descriptive. Since we are not using the `checkoutProgress.description`, in `checkout.ts` -> `checkoutBranch`, we can set the description of the `checkoutProgress` to 'Switching to Branch` and in `checkoutCommit` to `Checking out a Commit`. Then this bit of logic can use the `checkoutProgress.description` instead.